assets/scss/templates/_reportes.scss

```diff
@@ -171,3 +171,46 @@
     }
   }
 }
+
+/* Top ranking */
+.wrapper-ranking {
+  max-width: $dp-wrapper-max-width;
+  margin: 25px auto;
+  font-family: $dp-font-family;
+
+  .reports-box {
+    @include dp-box($dp-color-sylver, 100%, 100%, 100%);
+    position: relative;
+    padding: 0;
+  }
+  .title-ranking {
+    padding: 40px 30px;
+    display: inline-block;
+    font-size: $dp-font-size-small;
+    color: $dp-color-darkgrey;
+    line-height: 18px;
+  }
+  .page-ranking--item {
+    border-top: 1px solid $dp-color-sylver;
+    border-bottom: 1px solid $dp-color-sylver;
+    padding: 42px 30px;
+    background: #f6f6f6;
+    margin-bottom: 5px;
+    min-height: 208px;
+
+    .text-ranking {
+      margin: 16px 0;
```

Comment:
Should it not use any space variable?

(Maybe there are more)

Reply:
Ok sorry, forget me. I correct